src/containers/Register/UserSlice.ts

```diff
@@ -38,6 +39,7 @@ const initialState: IUserState = {
     loginSuccessValue: false,
     loginErrorValue: null,
     token: loadValidToken(),
+    email: localStorage.getItem("email"),
```

Comment:
No need for the email to be in the local storage. In case it gets changed, you need to keep track of it and there is really no point it keeping it in the local storage as it doesn't differ from other data really.

Reply:
Oh, I see why you meant local storage.. We have on backend an endpoint that returns user data so you can fetch it always - `user/account`. This is much better approach as it can be more data in the user account (name, nationality etc.) that should be loaded.